tap_stripe/__init__.py

```diff
@@ -428,6 +437,16 @@ def get_bookmark_for_stream(stream_name, replication_key):
             int(utils.strptime_to_utc(Context.config["start_date"]).timestamp())
     return stream_bookmark
 
+def get_bookmark_for_sub_stream(stream_name):
+    sub_stream_name = stream_name
+    stream_name = PARENT_STREAM_MAP[stream_name]
+    stream_metadata = metadata.to_map(Context.get_catalog_entry(stream_name)['metadata'])
+
+    replication_key = metadata.get(stream_metadata, (), 'valid-replication-keys')[0]
```

Comment:
Use `STREAM_REPLICATION_KEY` map to get replication_key of parent stream directly rather than retrieving replication key after retrieving metadata.

Reply:
Updated the code